package.json

```diff
@@ -46,6 +46,12 @@
     "msw": "^0.35.0",
     "ts-jest": "^26.4.4"
   },
+  "dependencies": {
+    "@web3-ui/hooks": "0.4.0"
+  },
+  "peerDependencies": {
```

Comment:
It's already in the regular deps, we put it here only if we require the user to manually install the package in question

Reply:
I tried to solve the failing Ci step by adding this. Unfortunately, it doesn't solve that issue. Have you an idea what to do?

Reply:
Because you are importing from `src`